pokerpoll/pokerpoll.py

```diff
@@ -1,6 +1,6 @@
 import discord
 from discord.ext import commands
-from datetime import datetime as dt
+from datetime import datetime, timedelta as dt, tdelta
```

Comment:
seperate these or just import the full datetime module.
personally i think obfuscating the meaning of the original naming (dt, tdelta) is bad too